src/Symfony/Bundle/SecurityBundle/Tests/Functional/UserPasswordEncoderCommandTest.php

```diff
@@ -85,6 +86,28 @@ public function testEncodePasswordPbkdf2()
         $this->assertTrue($encoder->isPasswordValid($hash, 'password', $salt));
     }
 
+    public function testEncodePasswordArgon2i()
+    {
+        if (PHP_VERSION_ID < 70200 && !extension_loaded('libsodium')) {
+            $this->markTestSkipped('Argon2i algorithm not available.');
+
+            return;
```

Comment:
useless. ``markTestSkipped`` is throwing an exception to interrupt the execution of the test